### PR TITLE
Add AWS EC2 commands

### PR DESCRIPTION
**Pull request by satyarohith**
The following commands will be implemented:
- [x] `ec2_list`
- [x] `ec2_reboot`
- [x] `ec2_status`

### CODE REVIEW

File: aws/ec2/ec2_status.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * @return {Promise<SlackBodyType>} Response body
+ */
+async function _command(params, commandText, secrets = {}) {
+  const {awsAccessKey, awsSecretKey, awsRegion} = secrets;
+
+  if (!awsAccessKey || !awsSecretKey || !awsRegion) {
+    return {
+      response_type: 'ephemeral',
+      text:
+        'You must create secrets for `awsAccessKey`, `awsSecretKey` and `awsRegion` to use this command '
```
Comment (niks3089) [inline aws/ec2/ec2_status.js]: .. Using `/nc secret_create` might be more verbose

File: aws/ec2/ec2_status.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    };
+  }
+
+  const {id: instanceId} = params;
+  const result = [];
+  const aws = require('aws-sdk');
```
Comment (niks3089) [inline aws/ec2/ec2_status.js:21]: Move this at the top of the file

File: aws/ec2/ec2_status.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  const ec2 = new aws.EC2({
+    apiVersion: '2016-11-15',
+    accessKeyId: awsAccessKey,
+    secretAccessKey: awsSecretKey,
+    region: awsRegion
+  });
+
+  try {
+    const {promisify} = require('util');
```
Comment (niks3089) [inline aws/ec2/ec2_status.js:30]: Same comment as above

**Review by niks3089 — CHANGES_REQUESTED** [review]

File: aws/ec2/ec2_status.js (hunk 2 above)
Comment (satyarohith) [reply aws/ec2/ec2_status.js:21]: Lazy loading the module here.

**Pull request merged**